### PR TITLE
Content: Create descriptions for each recent program for Programs page

### LINKED ISSUE #72: Content: Create descriptions for each recent program for Programs page
As a user,
I need a brief description for  About Our Recent Programs section of the Programs Page
so that as a newcomer I understand each recent program.
The About Our Recent Programs should contain the  program title , program description and  program eligibility for participants for each program regards the AnitaB.org 

**Mocks-**
programs (2)

**Acceptance criteria** : the PR must contain the following programs
- [ ] Google Summber of Code
- [ ] Outreachy 
- [ ] Hacktoberfest

**Definition of Done**
- [ ] All of the required items are completed.
- [ ]  Approval by 1 mentor.

Estimation
3 hours

### PR DESCRIPTION
**Pull request by nandini45**
### Description
A brief description for About Our Recent Programs section of the Programs Page
so that a newcomer can understand each recent program.
Fixes #72 

### Type of Change:
- Documentation

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by annabauza** [comment]
my mistake @nandini45 I need one more correction

File: src/content/events_description.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{ 
+  "Event1" : [
```
Comment (annabauza) [inline src/content/events_description.json:2]: replace   
```
"Event1" : [
		{ 
```
with `["event": {

Comment (annabauza) [reply src/content/events_description.json:2]:
```
{
  "cards": [
    {
      "title": "Open Source Hackday",
      "content": [
        {
          "par": "Open Source Hack is a 1 Month long open source program by AnitaB.org Open Source Community. This program aims to help the participants contribute to various Open Source Projects. Top participants (maximum merged PRs) will get exciting coupons and letters of appreciation from the organization."
        },
        {
          "par": " The main aim of this events is to help people get involved with Open Source and most importantly help people overcome the “fear” of doing their first contributions."
        },
        {
          "header": "Eligibility Criteria"
        },
        {
          "par": "Open Source Hack organized by AnitaB.org is open for all (students and working professionals). It provides a platform to all the participants to showcase their skills in an international Open Source Program. The program starts at the end of July and continues for a time lapse of 1 month."
        }
      ]
    },
    {
      "title": "oogle summer of code",
      "content": [
        {
          "par": "The Google Summer of Code, often abbreviated to GSoC, is an international annual program in which Google awards stipends to students who successfully complete a free and open-source software coding project during the summer. It focuses on bringing more student developers into open source software development. "
        },
        {
          "par": "Students work with an open source organization on a 3 month programming project during their break from school. The program is open to all university students aged 18 or over. As a part of Google Summer of Code, student participants are paired with a mentor from the participating organizations, gaining exposure to real-world software development and techniques "
        },
        {
          "header": "Eligibility Criteria"
        },
        {
          "par": "Must be at least 18 years old at time of registration. \nMust be enrolled in or accepted into an accredited institution including (but not necessarily limited to) colleges, universities, masters programs, PhD programs and undergraduate programs as of the GSoC Student Acceptance Date. \nStudents who wish to apply for acceptance into the Program must submit Proof of Enrollment and a Project Proposal."
        },
        {
          "par": "The program starts at the end of Feburary and then the students submit their proposals by the end of March. \nFurthermore the accepted students start bonding with the community and working on their project starting from June till the end of August."
        }
      ]
    }
  ]
}
```

Comment (annabauza) [reply src/content/events_description.json:2]: @nandini45 this would be better I think ☝️

Comment (nandini45) [reply src/content/events_description.json:2]: ok @annabauza  i will change the code

**Comment by nandini45** [comment]
@annabauza  i did the changes..
 please review it.

**Pull request merged**